spring-server-generatedUpdated/src/main/java/io/swagger/api/BookApiController.java

```diff
@@ -37,33 +35,75 @@ public BookApiController(ObjectMapper objectMapper, HttpServletRequest request)
         this.request = request;
     }
 
-    public ResponseEntity<Void> addBook(@ApiParam(value = "Book object that needs to be added to the store" ,required=true )  @Valid @RequestBody Book body) {
+    public ResponseEntity<Void> addBook(@ApiParam(value = "Book object that needs to be added to the store", required = true) @Valid @RequestBody Book body) {
         String accept = request.getHeader("Accept");
-        return new ResponseEntity<Void>(HttpStatus.NOT_IMPLEMENTED);
+        System.out.println("Adding Book... " + body.getTitle());
+        bookService.addBook(body);
+        return new ResponseEntity(body, HttpStatus.OK);
     }
 
-    public ResponseEntity<Void> deleteBook(@ApiParam(value = "Book id to delete",required=true) @PathVariable("bookId") Long bookId,@ApiParam(value = "" ) @RequestHeader(value="api_key", required=false) String apiKey) {
+    public ResponseEntity<List<Book>> getBooks() {
+        System.out.println("Loading books...");
+        List<Book> books = bookService.findAll();
         String accept = request.getHeader("Accept");
-        return new ResponseEntity<Void>(HttpStatus.NOT_IMPLEMENTED);
+        if (accept != null && accept.contains("application/json")) {
+            try {
+                return new ResponseEntity(books, HttpStatus.OK);
+            } catch (Exception e) {
+                log.error("Couldn't serialize response for content type application/json", e);
+                return new ResponseEntity(HttpStatus.INTERNAL_SERVER_ERROR);
+            }
+        }
+
+        return new ResponseEntity<List<Book>>(HttpStatus.OK);
     }
 
-    public ResponseEntity<Book> getBookById(@ApiParam(value = "ID of book to return",required=true) @PathVariable("bookId") Long bookId) {
+    public ResponseEntity<Void> deleteBook(@ApiParam(value = "Book id to delete", required = true) @PathVariable("bookId") String isbn, @ApiParam(value = "") @RequestHeader(value = "api_key", required = false) String apiKey) {
         String accept = request.getHeader("Accept");
+        System.out.println("Deleting Book... " + isbn);
+        Book book = bookService.findById(isbn);
+        if (book == null) {
+            System.out.println("book not found");
```

Comment:
replace with log statment and dont forget to tadd the id of the book so as t . is easier to follow in the logs